assets/js/src/newsletter_editor/components/content.js

```diff
@@ -63,6 +63,21 @@ Module.findModels = function findModels(predicate) {
   return _.filter(blocks, predicate);
 };
 
+Module.renderContent = function renderContent(content) {
+  if (App._contentContainerView) {
+    App._contentContainer.destroy();
+  }
+  if (App._contentContainerView) {
```

Comment:
Both IFs check for `_contentContainerView` but one destroys `_contentContainer` and other one `_contentContainerView`. Is that intentional?

Reply:
That's a typo, fixed